app/messaging/fetch-application.js

```diff
@@ -0,0 +1,18 @@
+const util = require('util')
+const { get } = require('../repositories/application-repository')
+const sendMessage = require('../messaging/send-message')
+const { fetchApplicationResponseMsgType, applicationResponseQueue } = require('../config')
+
+const fetchApplication = async (message) => {
+  try {
+    const msgBody = message.body
+    console.log('received application fetch request', util.inspect(msgBody, false, null, true))
+    const application = await get(msgBody.application)
```

Comment:
`msgBody.application` will need updating to `bookingReference`, if the term is agreed on.